shared/src/commonMain/kotlin/com/bunbeauty/shared/data/mapper/payment/PaymentMethodMapper.kt

```diff
@@ -20,10 +20,10 @@ class PaymentMethodMapper {
         )
     }
 
-    fun toPaymentMethod(paymentMethodEntity: PaymentMethodEntity): PaymentMethod? {
-        val name = PaymentMethodName.values().find { paymentMethodName ->
+    fun toPaymentMethod(paymentMethodEntity: PaymentMethodEntity): PaymentMethod {
```

Comment:
Why do we need UNKNOWN?
If we do so then we will show "-" to users
Let's leave as it is
Just return `null` and filter null values